src/store/migrations/1.js

```diff
@@ -1,10 +1,10 @@
-import { randomHex } from '../../modules/encryption/crypto';
+import randomHex from '@/modules/encryption/crypto';
 
 export default {
   number: 1,
   migration(state) {
     state.Importers.importers = state.Importers.importers.map((importer) => {
-      importer.id = importer.id || randomHex();
+      importer.id = randomHex();
```

Comment:
I'm tired. Need to think if there is a problem with replacing an existed `id` with a new one (if there is such case at all).